ladybug_charts/to_figure.py

```diff
@@ -117,6 +117,11 @@ def heat_map(
         )
     )
 
+    if dtick is not None:
+        ticktext = list(dtick.values())
+        tickval = list(dtick.keys())
+        fig.update_traces(colorbar_tickmode = 'array', colorbar_ticktext = ticktext, colorbar_tickvals = tickval, selector=dict(type='heatmap'))
```

Comment:
@anujm1ttal, we follow PEP8 for code styling. You can install one of the plugins to automatically check those for you.

```suggestion
    if dtick is not None:
        ticktext = list(dtick.values())
        tickval = list(dtick.keys())
        fig.update_traces(
            colorbar_tickmode ='array', colorbar_ticktext=ticktext,
            colorbar_tickvals=tickval, selector=dict(type='heatmap'))
```

Reply:
got it. thanks. will do that now 👍🏼